React/Base/RCTUIKit.h

```diff
@@ -425,6 +425,7 @@ CGPathRef UIBezierPathCreateCGPathRef(UIBezierPath *path);
 @property (nonatomic, assign) CGFloat zoomScale;
 @property (nonatomic, assign) BOOL alwaysBounceHorizontal;
 @property (nonatomic, assign) BOOL alwaysBounceVertical;
+@property (nonatomic, assign) BOOL enableFocusRing;
```

Comment:
Add a space between this and the other props so that it's clear the comment "// UIScrollView properties missing in NSScrollView" doesn't apply? And better yet, add a "// macOS specific properties" too? =)